client/landing/gutenboarding/onboarding-block/acquire-intent/site-title.tsx

```diff
@@ -17,9 +17,10 @@ import { Step, usePath } from '../../path';
 
 interface Props {
 	isVisible: boolean;
+	isMobile: boolean; // needed as a prop to be defined when component mounts
```

Comment:
We can leave this `undefined` as well:
```suggestion
	isMobile?: boolean; // needed as a prop to be defined when component mounts
```